locator/abstract_replication_strategy.cc

```diff
@@ -68,6 +68,17 @@ inet_address_vector_replica_set abstract_replication_strategy::get_natural_endpo
     return res->second;
 }
 
+stop_iteration abstract_replication_strategy::for_each_natural_endpoint_until(const noncopyable_function<stop_iteration(const inet_address&)>& func, const token& search_token, const effective_replication_map& erm) const {
```

Comment:
It's nicer to put the function as the last parameter, so there are no extra arguments past the lambda.